packages/ibm-products/previewer/codePreviewer.tsx

```diff
@@ -167,3 +199,22 @@ const removeUnknownComponents = (storyCode, unknownComponents) => {
   cleanedCode = cleanedCode.replace(closingTagPattern, '');
   return cleanedCode;
 };
+
+function detectReactHooks(sanitizedCode: string): string[] {
+  const hooksToCheck = [
+    'useState',
```

Comment:
We might want to expand this a bit further but for most stories these should cover all the bases.